staging/src/k8s.io/apimachinery/pkg/util/validation/field/errors_test.go

```diff
@@ -301,3 +249,438 @@ func TestErrorListRemoveCoveredByDeclarative(t *testing.T) {
 		}
 	}
 }
+
+func TestErrorFormatting(t *testing.T) {
```

Comment:
As we move from checking details strings to checking origins, we lose "indirect" test coverage for error formatting. That makes these unit tests important as they (eventually) will be the only place where we actually see the effect of changing error formatting.

Coverage of k8s.io/apimachinery/pkg/util/validation/ with this PR is at `62.4%`. Not too bad, but probably could be improved in a follow-up (good first issue/help wanted ...).

Also, line coverage can be misleading: we need coverage of various edge cases with different values that share the same lines of code.

Having said that, this list seems good enough for now.